reference/3.0/Microsoft.PowerShell.Utility/Select-Object.md

```diff
@@ -100,71 +125,136 @@ ati2evxx     Thursday
 
 This command gets the name and start day of the processes running on a computer.
 
-The command uses the Get-Process cmdlet to get the processes on the computer.
-It passes the processes to the **Select-Object** cmdlet, which creates objects that have only the **ProcessName** parameter and a calculated property named "Start Day." The "Start Day" property is added by using a hash table with Name and Expression keys.
-The value of the Expression key is a script blocks that gets the StartTime property of each process and the DayofWeek property of the StartTime.
-### Example 5
+The command uses the `Get-Process` cmdlet to get the processes on the computer.
+It passes the processes to the `Select-Object` cmdlet, which creates objects that have only the *ProcessName* parameter and a calculated property named Start Day.
+The Start Day property is added by using a hash table with Name and Expression keys.
+The value of the Expression key is a script blocks that gets the **StartTime** property of each process and the **DayofWeek** property of the StartTime.
+
+### Example 5: Select unique characters from an array
+
+```powershell
+"a","b","c","a","a","a" | Select-Object -Unique
 ```
-PS C:\> "a","b","c","a","a","a" | Select-Object -Unique
 
+```output
 a
 b
 c
 ```
 
-This command uses the Unique parameter of Select-Object to get unique characters from an array of characters.
-### Example 6
+This command uses the Unique parameter of `Select-Object` to get unique characters from an array of characters.
+
+### Example 6: Select newest and oldest events in the event log
+
+```powershell
+$a = Get-EventLog -LogName "Windows PowerShell"
 ```
-PS C:\> $a = Get-Eventlog -Log "Windows PowerShell"
-$a | select-object -index 0, ($a.count - 1)
+
+```powershell
```

Comment:
Merge the 2 code blocks. Make the case match for $a.  Make this change in all versions.